app/frontend/src/app/layout.tsx

```diff
@@ -0,0 +1,67 @@
+import type { Metadata } from "next";
+import { Inter } from "next/font/google";
+import "./globals.css";
+import localFont from 'next/font/local'
+import Image from "next/image";
+import { Avatar, AvatarFallback, AvatarImage } from "@/components/ui/avatar" /*Importinh avatar from shadcn */
+import Footerwave from "../../public/Footer_Wave.svg"; /*Imporrting the wave svg */
+
+const inter = Inter({ subsets: ["latin"] });
+const Virgil = localFont({ src: './Virgil.woff2' }) /*Importing the virgol font */
+
+export const metadata: Metadata = {
+  title: "AcademyX",
+  description: "Generated by create next app",
+};
+
+export default function RootLayout({
+  children,
+}: Readonly<{
+  children: React.ReactNode;
+}>) {
+  return (
+    <html lang="en">
+      <body >
+        <main className={Virgil.className}>{/*Virgil is the font used for the academy x logo */}
+        <header className="w-screen h-1/6 flex justify-between items-center border-academy bg-black  border-2 rounded-b-sm border-t-0">
+          <div className=" p-2">{/*This div will have the hamburger menu, which i have yet not worked on */}
+
+          </div>
+          <div className=" p-2 flex justify-start text-lg">{/*This div has the AcademyX text aka our logo*/}
+          <p    className=" text-white">Academy</p>
+          <p    className=" text-academy">X</p>
+          </div>
+          <div className=" p-2">{/*This div has the avatar properties */}
+            <Avatar>
+              <AvatarImage src="https://github.com/shadcn.png" />
+              <AvatarFallback>CN</AvatarFallback>
```

Comment:
Let the fallback take props too. It will be a nickname stored in the DB as the users initials.